apps/files/tests/test_utils_.py

```diff
@@ -133,8 +209,8 @@ def test_instanciate_without_data(self):
         with NamedTemporaryFile() as file_:
             file_.write(json.dumps(data))
             file_.flush()
-            mje = ManifestJSONExtractor(file_.name)
-            assert mje.data == data
+            pje = PackageJSONExtractor(file_.name)
+            assert pje.data == data
```

Comment:
I think the fix was in a commit in a PR that was rebased... and lost at that moment. That's the only explanation I can think of.